app/assets/stylesheets/alchemy/_variables.scss

```diff
@@ -136,7 +136,7 @@ $table-row-even-background-color: $white !default;
 $table-row-odd-background-color: rgba($white, 0.5) !default;
 $table-row-hover-color: rgba($light_yellow, 0.5) !default;
 
-$elements-window-width: 400px !default;
+$elements-window-width: 450px !default;
```

Comment:
Ideally we would make this responsive. Hardcoding a large value is a bad idea for smaller screens.